routers/web/user/profile.go

```diff
@@ -318,6 +319,15 @@ func Action(ctx *context.Context) {
 		return
 	}
 
-	shared_user.PrepareContextForProfileBigAvatar(ctx)
-	ctx.HTML(http.StatusOK, tplProfileBigAvatar)
+	if ctx.ContextUser.IsIndividual() {
+		shared_user.PrepareContextForProfileBigAvatar(ctx)
+		ctx.HTML(http.StatusOK, tplProfileBigAvatar)
+		return
+	} else if ctx.ContextUser.IsOrganization() {
+		ctx.Data["IsFollowing"] = ctx.Doer != nil && user_model.IsFollowing(ctx, ctx.Doer.ID, ctx.ContextUser.ID)
+		ctx.HTML(http.StatusOK, tplFollowUnfollow)
+		return
+	}
+	log.Error("Failed to apply action %q: unsupport context user type: %s", ctx.FormString("action"), ctx.ContextUser.Type)
+	ctx.Error(http.StatusBadRequest, fmt.Sprintf("Action %q failed", ctx.FormString("action")))
```

Comment:
Why doesn't the same action apply as for users?
Does the org not use HTMX on that page?
If no, then I'd suggest adding it there as well.
And in the other case, I don't see reasons to deviate from the user behavior…

Reply:
user profile has the count of followers, that should also be refreshed. But org home page doesn’t.

Reply:
Makes sense, unfortunately.
Ultimately, I think the behavior should be unified, but okay, I can accept that for now.

Reply:
This also confused me.
You can check the comment: https://github.com/go-gitea/gitea/pull/29005#issuecomment-1920669635

Reply:
I can take a stab at this after this PR is merged